Tests/RespondersTests/RouteResponderTest.swift

```diff
@@ -98,7 +98,7 @@ class RouteResponderTest: XCTestCase {
 
       let expected = "GET /someRoute HTTP/1.1"
 
-      XCTAssertEqual(response.body!, expected.toBytes)
+      XCTAssertEqual(response.body!.toBytes, expected.toBytes)
```

Comment:
I'm not sure if you're going to be able to do that...you'll probably have to convert one or the other and then compare. If you change the response to the invalidRequest I'm guessing this fails?